end-to-end-tests/cypress.config.js

```diff
@@ -9,6 +9,6 @@ module.exports = defineConfig({
   e2e: {
     setupNodeEvents(on, config) {},
   },
-  retries: 1,
+  retries: 2,
```

Comment:
I really don't like this and will look into why the `fetch` is failing. However, no amount of waiting will ever fix the `fetch` call failing. I am conflicted on if this is going too far to get the tests to pass and am open to suggestions.

Reply:
What do the Docker logs show on runs where the `fetch` fails?